configs/train-large.yaml

```diff
@@ -1,7 +1,15 @@
 train_config:
   model_id_prefix: 'segmentation-model-large'
   dataset_id: 'dataset-large'
+  segmentation_model:
```

Comment:
@rak5216 check out the configs especially - make sure they jive with you

Reply:
looks great. do you want me to verify the changes work, or perhaps you've likely done that?

Reply:
I have but for decide sized changes like these (that touch many different files) it'd probably be worthwhile for you to test it too. I'm not sure we've talked about how to change/test a different branch -- if not, happy to talk through it tomorrow

Reply:
so you mean test by cloning your branch before merging to master?

Reply:
everything looks great, should I clone  your branch?

Reply:
new changes have been verified to work on my end @Josh-Joseph